client/v1/save_.py

```diff
@@ -27,13 +27,16 @@
 
 def multi_image_tarball(
     tag_to_image,
-    tar
+    tar,
+    precompress_layers=False,
```

Comment:
Since we pass in `tar`, it is conceivable to do this in a separate function.  I'm not sure if I'd find that preferable, I'm just trying to reason about alternatives and thinking out loud.